### PR TITLE
Josh test

### PR DESCRIPTION
**Pull request by JoshuaDH81**

### CODE REVIEW

File: public/css/desktop.css [hunk 1]
```diff
@@ -1,4 +1,4 @@
-@import url('https://fonts.googleapis.com/css2?family=Gabarito&family=Paytone+One&family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;1,400;1,500&family=Rubik:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,300;1,400;1,500;1,600;1,700;1,800&display=swap');
+@import url('https://fonts.googleapis.com/css2?family=Candal&family=Gabarito:wght@400;500;600;700;800;900&family=Paytone+One&display=swap');
```
Comment (JoshuaDH81) [inline public/css/desktop.css:1]: replaced the old fonts link as a new font added for footer

File: views/footer.ejs
Comment (JoshuaDH81) [inline views/footer.ejs:1]: Added the footer file

File: views/home.ejs
Comment (JoshuaDH81) [inline views/home.ejs:1]: fixed nav and added footer

File: views/login.ejs
Comment (JoshuaDH81) [inline views/login.ejs:1]: changed nav

File: views/register.ejs
Comment (JoshuaDH81) [inline views/register.ejs:1]: changed nav

File: views/profile.ejs
Comment (JoshuaDH81) [inline views/profile.ejs:1]: changed nav

**Review by BenBarkworth-SBG — APPROVED** [review]

**Pull request merged**